app/controllers/pages_controller.rb

```diff
@@ -26,6 +26,7 @@ def dataset
     @time ||= 'present'
     @year = Current.setting.end_year if @time == 'future'
     @year ||= @area.analysis_year
+    @scenario_id = Current.setting.api_session_id if params[:scenario]
```

Comment:
Another option here might be to change the parameters given when displaying the factsheet. Currently we have something like:

`/regions/nl?scenario=true&time=future`

Perhaps instead of `scenario=true` it could be `scenario=12345` (where 12345 is the ID of a scenario)? This has the advantage that the URL can be shared with others. Again not necessary to change it for this PR, but perhaps a nice-to-have if you do more work on the CO2 factsheet later?